data/hai/hai missions.txt

```diff
@@ -1658,3 +1658,140 @@ mission "Strider notification"
+			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
+			label normal
+			`	"We just want to let you know what happened," the officer says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
+			`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. They're not in trouble, as there's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
+			choice
+				`	"So, am I in trouble or not?"`
+					goto trouble
+				`	"Are my passengers going to be okay?"`
+			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though."`
+			label trouble
+				`"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
+				`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`
+					decline
+
+
+
+mission "Station Exploration Abandon Ship"
+	invisible
+	landing
+	source
+		attributes hai
+		not attributes uninhabited
+	to offer
+		has "Station Exploration 1: done"
+		not "Station Exploration 2: done"
+	on offer
+		fine 70000
+		conversation
+			`Upon landing, you are accosted by the local police on <origin>. Behind your ship, you see your passengers disembarking from another Hai ship. Some of them are crying, but they appear uninjured.`
+			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
+			choice
+				`	"Am I in trouble?"`
+					goto normal
+				`	"I'm innocent!"`
+					goto normal
+				`	(Run away.)`
+			`	Almost as soon as you turn around to run you feel a sharp pain in your side as your consciousness fades away.`
+			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
+			label normal
+			`	You can clearly tell that the officer has little patience for your antics. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries.`
```

Comment:
It seems to me that asking whether you are in trouble or not isn't really an antic, although the officer should still be short on patience about the question. I tried writing something in a suggestion but I wasn't happy with it.

Reply:
Added a branch for if you asked if you were in trouble.